web/src/main/java/com/microsoft/applicationinsights/web/internal/correlation/tracecontext/Tracestate.java

```diff
@@ -0,0 +1,46 @@
+package com.microsoft.applicationinsights.web.internal.correlation.tracecontext;
+
+import org.apache.http.annotation.Experimental;
+
+/**
+ * Class that represents Tracestate header based on
+ *
+ * @author Reiley Yang
+ * @author Dhaval Doshi
+ * @link https://github.com/w3c/trace-context/blob/master/trace_context/HTTP_HEADER_FORMAT.md
+ *
+ * Implementations can add vendor specific details here.
+ */
+@Experimental
```

Comment:
Is something looking for these annotations?

Reply:
No I have marked this just because, ultimately this classes would be part of tracecontext libraray under different package name. Therefore, I just want to ensure no one depends on it.